Javascript Course/Session snippets/Session 6/challenge session 6.js

```diff
@@ -0,0 +1,52 @@
+
+let inventario = []
+
+function agregarItem(nombre, cantidad, precio){
+    const item = {nombre, cantidad, precio};    
+    inventario.push(item)    
+}
+
+//#region AddItems
+agregarItem("Manzana", 10, 0.5);
+agregarItem("Plátano", 5, 0.3);
+agregarItem("Naranja", 8, 0.4);
+agregarItem("Sandia", 6, 0.8);
+//#endregion
+
+
+function verificarInventario(){
+    let header = "Inventario: \n   Fruta \t      Cantidad       Precio \n"
+    for (const fruta of inventario) {
```

Comment:
Inventario, ya es un array, CREO que puedes usar el [forEach ](https://www.w3schools.com/jsref/jsref_foreach.asp) en vez de un for pelon